project/Component/NefCommon/Models/NefPlaygroundURL.swift

```diff
@@ -45,6 +46,7 @@ public struct NefPlaygroundURL {
             case .log:           return "\(Component.nef.pathComponent)/\(rawValue)"
             case .build:         return "\(Component.nef.pathComponent)/\(rawValue)"
             case .fw:            return "\(Component.build.pathComponent)/\(rawValue)"
+            case .swiftmodules:  return "\(Component.build.pathComponent)/\(rawValue)"
```

Comment:
Why is this case needed? You already have `.fw` which has the same `pathComponent`.

Reply:
Yep, right now both have the same parent-path but with a different path:
1. `<build path>/fw`
2. `<build path>/swiftmodules`